wasm/wallet_test.go

```diff
@@ -0,0 +1,178 @@
+package wasm
+
+import (
+	"fmt"
+	"syscall/js"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"net/http/httptest"
+
+	"github.com/0chain/gosdk/core/logger"
+	"github.com/0chain/gosdk/zboxcore/blockchain"
+	"github.com/0chain/gosdk/zcncore"
+)
+
+var validMnemonic = "inside february piece turkey offer merry select combine tissue wave wet shift room afraid december gown mean brick speak grant gain become toy clown"
+
+var walletConfig = "{\"client_id\":\"9a566aa4f8e8c342fed97c8928040a21f21b8f574e5782c28568635ba9c75a85\",\"client_key\":\"40cd10039913ceabacf05a7c60e1ad69bb2964987bc50f77495e514dc451f907c3d8ebcdab20eedde9c8f39b9a1d66609a637352f318552fb69d4b3672516d1a\",\"keys\":[{\"public_key\":\"40cd10039913ceabacf05a7c60e1ad69bb2964987bc50f77495e514dc451f907c3d8ebcdab20eedde9c8f39b9a1d66609a637352f318552fb69d4b3672516d1a\",\"private_key\":\"a3a88aad5d89cec28c6e37c2925560ce160ac14d2cdcf4a4654b2bb358fe7514\"}],\"mnemonics\":\"inside february piece turkey offer merry select combine tissue wave wet shift room afraid december gown mean brick speak grant gain become toy clown\",\"version\":\"1.0\",\"date_created\":\"2021-05-21 17:32:29.484657 +0545 +0545 m=+0.072791323\"}"
+
+var storageConfig = fmt.Sprintf("{\"wallet\":%s,\"signature_scheme\":\"bls0chain\"}", walletConfig)
+
+var server *httptest.Server
+var miner *httptest.Server
+var sharder *httptest.Server
+var Logger logger.Logger
+
+func TestWasmWallet(t *testing.T) {
+	Logger.Info("Testing WASM version of Zcncore.Wallet")
+	defer server.Close()
+	t.Run("Setting All Configuration", func(t *testing.T) {
+		TestAllConfig(t)
```

Comment:
The idea of subtests is not to call another function, but to reduce the amount of functions. Test code should be inside subtest

Reply:
I fixed it.